core/plugin_api.php

```diff
@@ -185,21 +185,17 @@ function plugin_route_group( $p_base_name = null ) {
 }
 
 /**
- * Return a path to a plugin file.
- * @param string $p_filename  File name.
- * @param string $p_base_name Plugin base name.
- * @return mixed File path or false if FNF
+ * Return a path to a plugin file(s).
+ * @param string $p_filename  File name (if it is not set, the function will return a directory).
```

Comment:
*a directory* is vague. I suggest to rephrase: 
`File name; if not set, the function will return the plugin's files directory.`